core/modules/wsched/BlendScheduler.cc

```diff
@@ -67,12 +67,25 @@ BlendScheduler* dbgBlendScheduler=nullptr; ///< A symbol for gdb
 ////////////////////////////////////////////////////////////////////////
 // class BlendScheduler
 ////////////////////////////////////////////////////////////////////////
-BlendScheduler::BlendScheduler(std::shared_ptr<GroupScheduler> group,
-                               std::shared_ptr<ScanScheduler> scan)
-    : _group{group}, _scan{scan}
+BlendScheduler::BlendScheduler(std::string const& name,
+                               int schedMaxThreads,
+                               std::shared_ptr<GroupScheduler> const& group,
+                               std::shared_ptr<ScanScheduler> const& scanFast,
+                               std::shared_ptr<ScanScheduler> const& scanMedium,
+                               std::shared_ptr<ScanScheduler> const& scanSlow)
```

Comment:
It might be nice and it probably wouldn't be hard to do, but I'm not sure the functionality would be used. I'm not sure it's worth the effort at this point as I'd like to see how well this works on the cluster before changing it. It's possible that each ScanScheduler will need some more special treatment and it's easier to do that when they aren't completely generic.